dali/python/nvidia/dali/external_source.py

```diff
@@ -159,11 +185,16 @@ class ExternalSource():
 
     The source is polled for data (via a call ``source()`` or ``next(source)``)
     when the pipeline needs input for the next iteration. Depending on the value of ``num_outputs``,
-    the source can supply one or more data batches. If ``num_outputs`` is not set, the ``source``
-    is expected to return one batch. If this value is specified, the data is expected to a be tuple,
-    or list, where each element corresponds to respective return value of the external_source.
-    If the source is a callable that accepts a positional argument, it is assumed to be the current
+    the source can supply one or more data items. The data item can be a whole batch (default) or
```

Comment:
As we previously talked I would put the list of supported types here as well, I bet no one check fee_inputs looking for it.

Reply:
Done.